.envdefault

```diff
@@ -1,17 +1,14 @@
 # SERVING PARAMETERS
 SERVICE_MODE=http
-MODEL_TYPE=lin
-
-# HTTP PARAMETERS
-ENABLE_STREAMING=true
+LANGUAGE=fr
```

Comment:
I realized that `LANGUAGE` is already a standard environment variable.
Wondering what could be a better name here... ASR_LANGUAGE?
@damienlaine any idea?